### ISSUE TITLE
LD_PRELOAD cannot be preloaded && undefined symbol

### ISSUE THREAD

**Issue by LingyuZhuFin** [yuxng/DA-RNN#12]
Hi,
Q1. when I train and test RNN with rgbd data
./experiments/scripts/rgbd_scene_multi_*.sh 0
I got the error of "ERROR: ld.so: object '/usr/lib/libtcmalloc.so.4' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored." 

Q2. Encountered this error of "tensorflow.python.framework.errors_impl.NotFoundError: /home/path/DA-RNN-master/tools/../lib/backprojecting_layer/backprojecting.so: undefined symbol: _ZN10tensorflow16KernelDefBuilderD2Ev
"
How you guys cope with this problem? Thanks

**Comment by LingyuZhuFin** [comment]
There is no libtcmalloc.so.4 in the path /usr/lib/, and couldn't find this libtcmalloc.so.4  package online(seems like libtcmalloc_minimal.so.4 is avaliable online).
And have no idea with the Q2

**Comment by kevinkit** [comment]
You may need to add a symbolic link to your numpy installation from within 
  
e.g.:
```Shell
cd /usr/local/include
ln -s /usr/local/lib/python2.7/site-packages/numpy/core/include/numpy numpy
```
 

I think that #10 mentions how to change sth for numpy, which will do the same trick. 


Q2, have you checked the version of CUDA,cuDNN and Tensorflow? See #2 


also running make -j32 and right after that make again sometimes fixes the problem .....

**Comment by kevinkit** [comment]
You may need to execute it with sudo

**Comment by LingyuZhuFin** [comment]
I tried this, and still not work. My system is Ubuntu 16.04. Seems like the dependencies version of Kniectfusion really matters. I also test with Ubuntu 14.04. There are always some errors exit.

Anyway, thank you very much.

**Comment by kevinkit** [comment]
Yes, they are crucial - most of the time it is pangolin messing up. 

However, since your second error comes from Tensorflow you should check if you have the correct tensorflow version (1.2.0) and that tensorflow-gpu 


you can find out by

`pip list | grep tensorflow-gpu`


if it does not find tensorflow-gpu , but this command works:

`pip list | grep tensorflow`

You have installed tensorflow with no gpu support


Nevertheless, you may have both and tensorflow gets confused. To check if it is installed you can try to run this script:

[TF-GPU](https://github.com/kevinkit/TensorTut/blob/master/Lesson0/Tfgpu.py)


It should show you your kind of GPU and the result of a matrix multiplication.


You must have a NVIDIA-CUDA capable GPU (compute capability > 5) with at least 8 GB memory.

**Comment by yuxng** [comment]
sudo apt-get install google-perftools

**Comment by andjsmile** [comment]
Hi,I have the same problem with you, do you resolve it ? How to do it?
